scss/_navbar.scss

```diff
@@ -28,13 +28,13 @@
   --#{$prefix}navbar-nav-font-size: #{$font-size-base}; // Boosted mod
   --#{$prefix}navbar-nav-line-height: #{$line-height-base}; // Boosted mod
   --#{$prefix}navbar-nav-letter-spacing: #{$letter-spacing-base}; // Boosted mod
-  --#{$prefix}navbar-nav-link-padding-y: #{$navbar-nav-link-padding-y}; // Boosted mod
-  --#{$prefix}navbar-nav-link-padding-x: #{$navbar-nav-link-padding-x-xs}; // Boosted mod
-  --#{$prefix}navbar-nav-icon-padding-y: #{$navbar-nav-icon-padding-y-xs}; // Boosted mod
-  --#{$prefix}navbar-nav-icon-padding-x: #{$navbar-nav-icon-padding-x-xs}; // Boosted mod
+  --#{$prefix}navbar-nav-link-margin-y: #{$navbar-nav-link-margin-y}; // Boosted mod
+  --#{$prefix}navbar-nav-link-margin-x: #{$navbar-nav-link-margin-x-xs}; // Boosted mod: margin over padding for focus
```

Comment:
Maybe put the reason for the Boosted mod on line 31? Or in single multiline block?

Reply:
Not quite sure since it's display in the documentation, so it might be unclear for people that are not used to this project. Any thoughts @julien-deramond ?

Reply:
Do you mean the "instead of" comment? If so, IMO it would be disturbing for folks here since it's displayed in our docs. However, if this comment can help using this CSS var, could be useful.